src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,34 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.9.0
+12 June, 2023
+
+### Deprecation Notice
+
+- Support for Kubernetes [1.23](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md) has been deprecated and will be removed in the next release.
+
+### Features
+
+- Kubernetes supported versions: [1.26](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.26.md), [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md) and [1.24](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md).
+- Upgraded the Okteto CLI version to [2.16.2](https://github.com/okteto/okteto/releases/tag/2.16.2)
+- Telemetry job is now launched as soon as the helm installation is completed.
+- Admins can now check when it was the last interaction a user had with Okteto in the Admin dashboard, under the Users section.
+- Support to access Okteto on `127.0.0.1`
+- Upgrade ingress nginx to 4.5 
+
+### Improvements
+
+- Frontend now loads faster with improved nginx compression
+
+### Bugfixes
+
+- Improve UI static assets management across releases
+- Helm installations are now uninstalled always from dev environments on destroy all operations
+- Any container in development mode is now deactivated as part of the sleep namespace operation
+- Variables defined at deploy time are now available on destroy operations
+- Fixed a bug where the container image layer manifest could get corrupted upon being uploaded to the Registry. This would cause Buildkit to return an "500 Internal Server Error" for related build requests.
```

Comment:
_"Fixed a bug"_ is implied by this being listed under _"Bugfixes"_ section already. Also, could this be articulated more clearly? Maybe something on these lines?

E.g. 
```suggestion
- Container image layer manifest sometimes got corrupted when being uploaded to the Builkdkit Registry, causing "500 Internal Server Error" on following build requests
```